lookout/style/format/analyzer.py

```diff
@@ -86,7 +87,7 @@ def __init__(self, model: FormatModel, url: str, config: MutableMapping[str, Any
 
     @with_changed_uasts_and_contents
     def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
-                data_service: DataService, changes: Iterator[Change], **data) -> List[Comment]:
+                data_service: DataService, changes: Iterable[Change], **data) -> List[Comment]:
```

Comment:
Nopes, it is an `Iterator`, you can only iterate it once.

@m09 another one bites the dust

Reply:
yeah, it is tricky, looks I am wrong.